<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceMeshWidget</class>
 <widget class="QWidget" name="SurfaceMeshWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
     </property>
     <property name="title">
      <string>Source Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_9">
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_24">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>HDF Voxel File</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFSDropLineEdit" name="m_InputFile">
          <property name="text">
           <string>*.h5voxel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="m_InputFileBtn">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>Scalars</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_ScalarsName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QFormLayout" name="formLayout_4">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
          </property>
          <property name="text">
           <string>X Res:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="m_XRes">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
          </property>
          <property name="text">
           <string>0.2500</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
          </property>
          <property name="text">
           <string>Y Res:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="m_YRes">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="m_ZMax_2">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
          </property>
          <property name="text">
           <string>Z Res:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="m_ZRes">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
          </property>
          <property name="text">
           <string>X Dim:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="m_XDim">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
          </property>
          <property name="text">
           <string>Y Dim:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="m_YDim">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
          </property>
          <property name="text">
           <string>Z Dim:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="m_ZDim">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QCheckBox" name="m_SmoothMesh">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Smooth Surface Mesh</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_10">
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_26">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Number of Iterations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_SmoothIterations">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>12</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="m_LockQuadPoints">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Lock Quad Points</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_27">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Write output file every </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_WriteOutputFileIncrement">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_28">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Iterations</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2" colspan="2">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>148</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
     </property>
     <property name="title">
      <string>Output Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QLabel" name="label_25">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>Output Directory:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFSDropLineEdit" name="m_OutputDir"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="m_OutputDirBtn">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>Output File Prefix:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_OutputFilePrefix">
          <property name="text">
           <string>SurfaceMesh_</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_60">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>Temporary Files:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="m_VisualizationVizFile">
          <property name="text">
           <string>Visualization.vtk</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>VTK PolyData:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="m_VtkOptionsBtn">
          <property name="text">
           <string>VTK Options</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="m_WriteSTLFiles">
          <property name="text">
           <string>STL Files</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="m_DeleteTempFiles">
          <property name="text">
           <string>Delete Temporary Files</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>STL Files:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="messageLabel">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Message:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>169</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QProgressBar" name="m_progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_GoBtn">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFSDropLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QFSDropLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
